content/webinars/gitops-with-gitlab-and-pulumi-2020-05-20/index.md

```diff
@@ -71,10 +72,6 @@ main:
 
 # The right hand side form section.
 form:
-    # GoToWebinar webinar key. This key allows us to register people for webinars via the
-    # HubSpot form.
-    gotowebinar_key: "1017103637425347085"
-
     # HubSpot form id.
-    hubspot_form_id: "ebc6064d-756c-4d04-ac63-31ad16780ab5"
+    hubspot_form_id: "87cc0632-52f1-444a-b41c-6775470f4ece"
```

Comment:
Does this need to be changed for anything in this PR?

Reply:
It doesn't need to exist. Will remove.